.changeset/seven-spies-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+Amounts are now hidden on Transaction events list for CHARGE_REQUST, CHARGE_SUCCESS and CHARGE_FAILURE events. These events don't support providing amounts and they were always displayed as 0, which was misleading.
```

Comment:
suggestion: This description contains why you did those changes, by I'd rather prefer a short description of what you did for example: `Stop showing Transactions events that returns 0 amount`